### PR TITLE
Document how to release

### PR DESCRIPTION
**Pull request by raboof**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,3 +7,9 @@ addSbtPlugin("com.lightbend.akka" % "sbt-paradox-akka" % "<version>")
 ```
 
 This plugin extends the Paradox sbt plugin with some default settings and sets the Akka Paradox theme.
+
+## Releasing
+
+When adding a tag, travis should automatically release to
```
Comment (bantonsson) [inline README.md]: Yes, any tag that contains 🚢 or 🐑 will trigger a release.

**Comment by ktoso** [comment]
Could we put it to RELEASING.md and add some more links akin to https://github.com/akka/akka-http/blob/master/RELEASING.md#releasing ?

Basically copy this:

```
- Go to https://bintray.com/akka/sbt-plugin-releases/sbt-paradox-akka and select the just released version
- Go to the Maven Central tab and sync with Sonatype
- Log in to Sonatype to Close the staging repository
```

To save at least a bit "looking for where to click on stuff"

**Comment by raboof** [comment]
I added some more details - we don't really need to sync to sonatype, jcenter should be enough right?

**Comment by ktoso** [comment]
Ah you're right since it's a plugin - yeah

**Pull request merged**